SPECS/userspace-rcu/userspace-rcu.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        user space RCU (read-copy-update)
 Name:           userspace-rcu
-Version:        0.10.1
-Release:        5%{?dist}
+Version:        0.13.0
```

Comment:
Small ask: can you see if the nss dependency is still needed here? I might be missing it, but I can't find any references to nss in the configuration script for this package.

Reply:
Yes, removing it still built it. Thanks Thomas

Reply:
Thanks!